svir/dialogs/drive_oq_engine_server_dialog.py

```diff
@@ -199,16 +199,9 @@ def refresh_calc_list(self):
                 item.setTextColor(row_txt_color)
                 self.calc_list_tbl.setItem(row, col, item)
             for col, action in enumerate(actions, len(selected_keys)):
-                # do not display 'Continue' button, if this is already a risk
-                # calculation or if the calculation is still incomplete
-                if action['label'] == 'Continue':
-                    if (calc['job_type'] == 'risk'
-                            or calc['status'] != 'complete'):
-                        continue
-                # do not display the button for outputs until calc is complete
-                elif action['label'] == 'Outputs':
-                    if calc['status'] != 'complete':
-                        continue
+                if (calc['status'] != 'complete' and
+                        action['label'] in ('Continue', 'Outputs')):
+                    continue
```

Comment:
Perhaps a comment like "display the Continue or Output buttons only if the computation is completed"